enhancements/20230327-harvester-seeder-integration.md

```diff
@@ -0,0 +1,227 @@
+# Harvester Seeder Embedded Mode Integration
+
+## Summary
+
+We have been running and developing [Seeder](github.com/harvester/seeder) for provisioning and managing the lifecycle of baremetal clusters in our internal labs.
+
+With version 1.2.0, we plan to introduce an embedded mode of Seeder, which will allow deployment of Seeder to an existing cluster via Harvester Addons.
+
+In embedded mode, the addon will enable end-users to define Inventory objects that map the Harvester nodes to the corresponding baremetal nodes. The baremetal interaction will be through IPMI and Redfish protocols.
+
+Once defined, seeder in embedded mode, will leverage the cluster event controllers to query underlying hardware and propagate that information to the corresponding node objects.
+
+In addition, the users will be able to perform underlying hardware power cycle operations via the Harvester UI.
+
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/2318
+
+## Motivation
+
+### Goals
+
+- Allow propagation of hardware information into Harvester.
+
+- Allow propagation of hardware events into Harvester events.
+
+- Allow users to power cycle hardware via Harvester.
+
+### Non-goals [optional]
+
+Provisioning of additional hardware.
+
+## Proposal
+
+### User Stories
+
+#### Discover hardware details via Harvester
+
+#### User Experience In Detail
```

Comment:
It's a bit weird that these two headlines have the same level, but the former has no content. Could we combine these two?